.travis.yml

```diff
@@ -5,7 +5,8 @@ services:
 
 language: go
 go:
- - 1.9
+ - "1.10"
```

Comment:
not sure this is the version of go thats being used to build the survey service, i'll just confirm which version is being used